src/java/org/apache/cassandra/service/StartupChecks.java

```diff
@@ -569,6 +586,14 @@ public FileVisitResult preVisitDirectory(Path dir, BasicFileAttributes attrs) th
                                                          "upgradesstables",
                                                          Joiner.on(",").join(invalid)));
 
+            if (!withIllegalGenId.isEmpty())
+                throw new StartupException(StartupException.ERR_WRONG_CONFIG,
+                                           "UUID generation identifiers are disabled but some sstables have been " +
+                                           "created with UUID generation identifier. You have to either delete those " +
+                                           "sstables (if you are ok with loosing that data on this node, and then let " +
+                                           "the repair bring them back) or enabled UUID based generation identifers in " +
```

Comment:
We are not sure that repair is going to be able to bring that data back, right?

Reply:
I'll replace it with:

```java
                throw new StartupException(StartupException.ERR_WRONG_CONFIG,
                                           "UUID generation identifiers are disabled but some sstables have been " +
                                           "created with UUID generation identifier. You have to either delete those " +
                                           "sstables or enabled UUID based generation identifers in cassandra.yaml " +
                                           "(enable_uuid_generation_identifiers). The list of affected sstables is: " +
                                           Joiner.on(", ").join(withIllegalGenId) + ". If you decide to delete sstables, " +
                                           "and have that data replicated over other healthy nodes, those will be brought" +
                                           "back during repair");
```